download-latest.sh

```diff
@@ -148,11 +148,18 @@ get_os() {
 get_archi() {
     architecture=$(uname -m)
     case "$architecture" in
-    'x86_64' | 'amd64' | 'arm64')
+    'x86_64' | 'amd64' )
         archi='amd64'
         ;;
+    'arm64')
+        if [ $os -eq 'macos' ]; then # MacOS M1
```

Comment:
```suggestion
        if [ $os = 'macos' ]; then # MacOS M1
```

I just checked and it seems like `-eq` is here to compare numbers (?) and `=` is used to compare strings.